opensrp-app/src/main/java/org/smartregister/AllConstants.java

```diff
@@ -98,6 +98,7 @@
 
     public static final String CAMPAIGNS = "CAMPAIGNS";
     public static final String OPERATIONAL_AREAS = "OPERATIONAL_AREAS";
+    public static final String CURRENT_OPERATIONAL_AREA = "CURRENT_OPERATIONAL_AREA";
```

Comment:
This is not needed Plans should be downloaded for all `OPERATIONAL_AREAS`

Reply:
Done.